### DISCUSSION TITLE
MicroSD card for Sense Version | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1152] [Q&A]
# MicroSD card for Sense Version | Seeed Studio Wiki

This tutorial describes how to use the microSD card and file system on the XIAO ESP32S3.

https://wiki.seeedstudio.com/xiao_esp32s3_sense_filesystem/

**Comment by embk** [comment]
Which lipo battery do you recommend to fit between the shield and the Xiao?

**Comment by O-Kalis** [comment]
Can you please correct the CS GPIO for SD Card on sense board being on GPIO21, because it really isnt, its connect to D2 (GPIO3) and it really f***ing sucks to look at the most reliable side for your board to see lie after a full day headache. And its gonna be another couple of hours of headache because i already counted on it. Previously i though that these XIAO boards are quite great, but after discovering that you cant use 6 out of 13 available pins if you want to use all its features makes me think what idiot make it like this. Have a nice day

**Reply by O-Kalis** [reply]
Well..., maybe I was bit too impatient and in schematic it really is connected to GPIO21 and GPIO3 is separated with not soldered resistor. And my problem was just with mixed up pins for MISO and MOSI.  /o\

**Reply by nfs0619** [reply]
Thank you for your feedback and follow-up clarification! We completely understand how frustrating debugging can be, and we're glad to hear you identified the actual issue.
We'll make an effort to clarify such resistor-connected designs more clearly in future documentation to avoid confusion.
If you have any further questions, feel free to reach out to our technical support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)**.
